bot/app/database/db_objects.py

```diff
@@ -230,4 +217,105 @@ def all_users(self):
         conn.close()
         curs.close()
 
-        return users
\ No newline at end of file
+        return users
+
+def get_next_event():
+
+    """Retrieves the next upcoming event from the db
+
+    Returns:
+        Event object
+    """
+
+    conn = sqlite3.connect(DB_NAME)
+    curs = conn.cursor()
+    
+    curs.execute("""select max(eventID) from Events""")
+    event_id = curs.fetchone()
+    
+    curs.execute("""select * from Events where eventID=?""", event_id)
+    temp = [item for item in curs.fetchone()]
+    del temp[0]
+
+    try:
+        next_event = Event(*temp)
+    
+        curs.close()
+        conn.close()  
+
+        return next_event
+    
+    except Exception as e:
+        logging.error(e)
+
+        curs.close()
+        conn.close() 
+
+        return None
```

Comment:
This can be improved and simplified:

```python
with sqlite3.connect(DB_NAME) as conn:
    with conn.cursor() as curs:
        curs.execute("""select max(eventID) from Events""")
        event_id = curs.fetchone()

        curs.execute("""select * from Events where eventID=?""", event_id)
        next_event = None
        try:
            next_event = Event(*curs.fetchone()[1:])
        except Exception as e:
            logging.error(e)
        
        return next_event
```

The problem with this approach (using `*args`) is that if there is some information mission (e.g. Event has no time or somethimg like that), you will get a `TypeError` exception:

```
TypeError: Event() missing 1 required positional argument
```

But I'm not sure how the `sqlite3` module handles this internally, do you know if there is no entry for that attribute, it will just become `None`, as in:

```
[1,2,None,3]
```

Or will the list just be shorter?

```
[1,2,3]
```

We should test this first, if it's the first case we're lucky and can just use the code above, if we have the second case we need to build some checks around that.

Reply:
there are only two positional args in `Event` (`name` and `date`) and one in `Artist` (`name`, which is conveniently the PK in the latter case and should be an AK in the former), though, the others i have defaulted to `None` on purpose since real world-entities are sometimes just missing those attributes, too. so i'm quite sure, that we'll have no type errors because every row _must_ have the positional arguments when being fetched (since i needed to put them in in the first place, when adding them via `app/database/db_update.py`